core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/GroupByRelationalTest.java

```diff
@@ -76,6 +76,20 @@ public void testValidGroupBy() throws Exception {
     Assert.assertEquals(3, selectExpressions.size());
   }
 
+  @Test
+  public void testMixedValidityGroupBy() {
+    GroupByConfig config = new GroupByConfig("profession",
+                                             "numEmployees: countif(*): condition(salary>100000)," +
+                                               "stddevSal: stddev(salary)," +
+                                               "maxSal: max(salary)");
```

Comment:
can we specify which expressions are valid for pushdown?

Reply:
They are specified in `functionSqlMap` in `GroupByAggregator`.